libs/windy-sounding/types/client/d.ts.files/alerts.d.ts

```
@@ -0,0 +1,114 @@
+import { AlertConditionType, CloudCoverage, AlertStatus, Direction, Weekday } from '@windy/userAlerts';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)